cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000076
-name: Median MS1 peak FWHM for peptides in RT quantile
-def: "Median of all MS1 peak widths at half maximum (FWHM) for all identified peptide, in seconds, in the respective identifications' quantiles ordered by retention time. The data is given in a tuple defining the quantiles, 4-tuple represents quartiles, 10-tuple deciles, etc." [PSI:QC]
-comment: Sharper peak widths (i.e. smaller values) indicate better chromatographic resolution in the given quantile.
-is_a: MS:4000004 ! n-tuple
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000086 ! full width at half-maximum
-relationship: has_relation MS:4000013 ! single run based
+name: ID ratio
+def: "The ratio of identified and recorded MS2 spectra at given FDR threshold." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units STATO:0000300 ! dimensionless ratio
 
 [Term]
 id: MS:4000077
-name: Area under TIC
-def: "The area under the total ion chromatogram." [PSI:QC]
+name: AA sequence length
+def: "The length of an amino acid sequence." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000235 ! total ion current chromatogram
-relationship: has_relation MS:4000013 ! single run based
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000078
-name: Area under TIC RT quantiles
-def: "The area under the total ion chromatogram of the retention time quantiles. Number of quantiles are given by the n-tuple." [PSI:QC]
-is_a: MS:4000004 ! n-tuple
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000235 ! total ion current chromatogram
-relationship: has_relation MS:4000013 ! single run based
+name: protein coverage table
+def: "The coverage of protein sequences from the peptide sequences identified. The table records the coverage itself (in percent), the protein accession (SHOULD correspond to the accession used in fasta document used for Sequence DB based identification), length of the protein (, and optionally if it is 'target' or 'decoy' entry and how many PSM can be attributed to this protein)." [PSI:QC]
+comment: The protein coverage can provide insight into the sensitivity of the instrument/identification/protocols used.
+is_a: MS:4000005 ! table
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_column MS:1001093 ! sequence coverage
+relationship: has_column MS:1000885 ! protein accession
+relationship: has_column PEFF:0001006 or term above ???
+relationship: has_optional_column MS:1001197 ! DB composition target+decoy ???
```

Comment:
i think this term only defines the state of the FASTA DB as a whole (e.g. its a combined db).
It does not define the allowed values, `target` and `decoy` AFAICT.